packages/ui/src/components/va-pagination/VaPagination.demo.vue

```diff
@@ -1,216 +1,376 @@
 <template>
-  <VbDemo>
+  <VbDemo class="va-pagination-demo">
     <VbCard>
       Current Page
       <input
-        type="number"
         v-model.number="activePage"
+        aria-hidden="true"
+        type="number"
       />
     </VbCard>
     <VbCard>
       <table class="table table-bordered">
         <tr>
           <th>Description</th>
-          <th>Va Pagination</th>
+          <th>VaPagination</th>
         </tr>
         <tr>
-          <td>Default Pagination</td>
```

Comment:
Pretty sure button should lose selection when it becomes active.

<img width="269" alt="image" src="https://user-images.githubusercontent.com/5394573/189231422-a229cdb2-4ac2-4c5e-ab14-7ebec4d386cd.png">

Reply:
Active button is `disabled` so it can't be focused. I can implement 'personal' disabled state for `VaPagination` active button with blackjack and focus but honestly that doesn't make sense for me.

Reply:
> Active button is disabled so it can't be focused

Why do we need that? Maybe let it be focusable?

Checked refs - everywhere buttons are focusable.

Reply:
Because current page button isn't interactive element. But i don't insist - will make it focusable.